Makefile

```diff
@@ -28,4 +29,36 @@ tidy:
 
 clean:
 	$(GO) clean -i ./...
-	rm -f vt
\ No newline at end of file
+	rm -f vt
+
+# Pretty: formats the code using gofumpt and goimports-reviser
+pretty: install-tools
+	@echo "Running formatting tools..."
+	@gofumpt -l -w . >/dev/null 2>&1 || true
```

Comment:
Is there a reason why not to use the same `gofmt` (or rather `goimports`) that `vitessio/vitess` uses?

Reply:
also @shlomi-noach, I think both of these are also supersets of what `vitessio/vitess` uses, i.e. the code will pass the scrutiny of `gofmt` and `goimports` with no change

Reply:
I think it's reasonable to use both new tools

Reply:
we can always try to use these two new tools on `vt` and get a better opinion after testing it for a while, helping decide if we want to use the same on vitess